openapi.yaml

```diff
@@ -255,6 +255,35 @@ paths:
           $ref: '#/components/responses/400'
         '500':
           $ref: '#/components/responses/500'
+  /servicenow/deductions:
+    post:
+      summary: Create payroll deductions
+      tags:
+        - deductions
+      description: Create payroll deductions
+      operationId: postDeductions
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeductionsPostBody'
+        description: Post body for payroll deductions
+        required: true
+      responses:
+        '200':
+          description: Successful response
```

Comment:
Since you are "creating" payroll deduction, a proper response code for this would be 201 - created.

Reply:
Got it, I'll update this. Thanks